packages/postgres-database/src/simcore_postgres_database/models/folders_v2.py

```diff
@@ -0,0 +1,85 @@
+import sqlalchemy as sa
+
+from ._common import (
+    column_created_datetime,
+    column_modified_datetime,
+    register_modified_datetime_auto_update_trigger,
+)
+from .base import metadata
+from .workspaces import workspaces
+
+folders_v2 = sa.Table(
+    "folders_v2",
+    metadata,
+    sa.Column(
+        "folder_id",
+        sa.BigInteger,
+        nullable=False,
+        autoincrement=True,
+        primary_key=True,
+        doc="Primary key",
+    ),
+    sa.Column(
+        "name",
```

Comment:
MINOR: This is a display name and not an identifier, right? We tend to mix both concepts under `name`. I would add that explicly in the column name

Reply:
discussed (for now I keep name and we can consolidate all at ones)